### ISSUE TITLE
What final parameters you use in model

### ISSUE THREAD

**Issue by jsheng0901** [peterkim95/attentive-reader#3]
Hi, recently, I'm do the same project as this. However, I didn't get good accuracy. I;m wondering what hyper parameters you use for training model like # units, learning rate, epochs. Thank you!
